frontend/src/ui-kit/components/Search/SearchResults.tsx

```
@@ -0,0 +1,177 @@
+  return (
+    <div
+      id={`search-result-${item.id}`}
+      role="option"
+      aria-selected={isSelected}
+      className={`
+        flex cursor-pointer items-center gap-3 px-4 py-3 transition-colors duration-100
+        ${isSelected ? 'bg-primary-50 dark:bg-primary-500/10' : 'hover:bg-surface-secondary'}
+      `}
+      onClick={handleClick}
+      onMouseEnter={onHover}
+    >
```

Biome (2.1.2)

[error] The HTML element with the interactive role "option" is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like "option".
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)